src/components/dashboard/Dashboard.vue

```diff
@@ -4,23 +4,24 @@
       <div class="container--value-panels-row">
```

Comment:
I believe there is a good reason to make a new dashboard for zkEVM, because as we add more zkEVM features to the Dashboard the code will became messy (a lot of `v-if` statements) and error prone

Reply:
Sounds good. Then @gtg7784 might want to set the logic at src/pages/Dashboard.vue to display the new dashboard for zkEVM.

Reply:
that's good
The reason is why I used the `v-if` is there's same component (the token supply graph). but separation is more clear and prevent messy, I'll separate logic to display the new dashboard.